src/Main.js

```diff
@@ -60,6 +61,8 @@ export const Main = () => {
       <CartModal />
       <OrderCreateModal />
       <OrderConfirmModal />
+
+      <Toaster />
```

Comment:
Should this be optional? Or part of the modal controller render function?

Reply:
it's already part of the `Controller` comp render, I'm using the same approach we always use, searching in children for a `viewId` of `toaster` and rendering it if it's there

Reply:
Yeah, but users will need to add `<Toaster />` for it to work. I'm talking about whether we should add it to be rendered by ModalController anyway. It's not a button or anything

Reply:
I completely agree with you, but I also wanted to respect the fact that the `Controller` comp is only a controller and doesn't force anything to be rendered, if the controller renders something as part of it's implementation then it's no longer a controller, so I preferred to respect the design over the inconvenience of adding the component